ulib/FStar.IndefiniteDescription.fst

```diff
@@ -28,22 +28,33 @@ module FStar.IndefiniteDescription
 
 (** The main axiom:
 
+    Given a proof of [p], we can obtain a witness for [p] in [GTot].
+*)
+assume
+val indefinite_unsquash (#p:Type) (x:squash p) : GTot p
```

Comment:
This is a nit, but to me `indefinite_unsquash` doesn't seem the right name for this. This is not reverting the `squash` type constructor, but in some way reverting `return_squash`. So how about `indefinite_unreturn_squash`? :)

Reply:
Or maybe something that doesn't sound like an inverse at all, like `indefinite_open_squash`?